<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1104</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>256</width>
        <height>0</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>254</width>
         <height>507</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>250</y>
          <width>258</width>
          <height>171</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>200</height>
         </size>
        </property>
        <property name="title">
         <string>Properties</string>
        </property>
        <widget class="QSlider" name="ISOValue">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>110</y>
           <width>84</width>
           <height>16</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="ISOLabel">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>110</y>
           <width>31</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>ISO</string>
         </property>
        </widget>
        <widget class="QLabel" name="opacityLabel">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>140</y>
           <width>61</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Opacity</string>
         </property>
        </widget>
        <widget class="QSlider" name="opacityValue">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>140</y>
           <width>84</width>
           <height>16</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QCheckBox" name="enabledCheckBox">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>40</y>
           <width>82</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Enabled</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="colorMapCheckBox">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>60</y>
           <width>91</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>ColorMap</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="continuousISOCheckBox">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>80</y>
           <width>141</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>ContinuousISO</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>70</y>
          <width>256</width>
          <height>170</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>256</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>128</width>
          <height>170</height>
         </size>
        </property>
        <property name="title">
         <string>ISO Surfaces</string>
        </property>
        <widget class="QListWidget" name="ISOSurfacesList">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>231</width>
           <height>81</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
        <widget class="QPushButton" name="addISOButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>130</y>
           <width>91</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Add Surface</string>
         </property>
        </widget>
        <widget class="QPushButton" name="deleteISOButton">
         <property name="geometry">
          <rect>
           <x>139</x>
           <y>130</y>
           <width>101</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Delete Surface</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>241</width>
          <height>61</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="2" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Current ISO</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="MyGlWindow" name="openGLWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1104</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_File"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyGlWindow</class>
   <extends>QOpenGLWidget</extends>
   <header>myglwindow.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
